### DISCUSSION TITLE
Allure report not generating in certain pipelines but generating in others without issue

### DISCUSSION THREAD

**Discussion by Seantracey** [allure-framework/allure2#1424] [Bug Reports] [answered]
Since migrating to azure cloud from our previous on premise azure environment. We migrated our pipelines and repositories, some of them are successfully generating an allure report while other’s aren’t even though there’s been no code changes. Is there any suggestions as to why this may be occurring?

**Comment by Avaddon-tfa** [comment]
Hello there! That's a tough issue to investigate without further details. Do you mind providing more details on non-working environments?

**Comment by baev** [comment, marked as answer]
Official Allure extension for Azure Pipelines are now available https://marketplace.visualstudio.com/items?itemName=qameta.allure-azure-pipelines

History isn't yet supported, but we'll get there! Please submit feedback to our [discussions](https://github.com/orgs/allure-framework/discussions)
